src/index.js

```diff
@@ -32,14 +32,16 @@ class Bootstrap extends EventEmitter {
   /**
    * Start emitting events.
    */
-  start () {
+  start (cb) {
     if (this._timer) {
       return
     }
 
     this._timer = setInterval(() => this._discoverBootstrapPeers(), this._interval)
 
     this._discoverBootstrapPeers()
+    
+    if (cb) cb()
```

Comment:
Use `nextTick()` for improving the performance of this call. 

Check [libp2p/js-libp2p-mdns/src/index.js#L59](https://github.com/libp2p/js-libp2p-mdns/blob/master/src/index.js#L59)

Also, you can do this instead: 
```js
cb && nextTick(() => cb())
```